source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnRenderer.h

```diff
@@ -9,6 +9,11 @@
 
 @interface ACRColumnRenderer:ACRBaseCardElementRenderer
 
+@property BOOL containerHasFillAlignment;
```

Comment:
can we use this check (column->GetVerticalContentAlignment() != VerticalContentAlignment::Top instead of adding this property?

Reply:
I'm using this property to signalize that at least one of the columns in the columnset where this column belongs to has center or bottom as their contentalignment property

---
In reply to: [199282460](https://github.com/Microsoft/AdaptiveCards/pull/1626#discussion_r199282460) [](ancestors = 199282460)